docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1975 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #81.        T, M_T, Σ_T = core.preprocess(T, m)
</pre><p>You should move <code>core.preprocess</code>  to the top! And then use <code>_prescrump</code>  instead of <code>prescrump</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
> And then use `_prescrump` instead of `prescrump`

To catch inf/nan by the help of `M_T` in `_prescrump`?

Reply:
<p>Yes, that happens automatically</p>

Reply:
will revise it in next set of commits.